### PR TITLE
implement disk task for ALICloud and fix typos

### PR DESCRIPTION
**Pull request by LilyFaFa**
Implement disk task for AlibabaCloud
This is part of #4127.

### CODE REVIEW

File: upup/pkg/fi/cloudup/alitasks/disk.go [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/cloudup/aliup"
+	"k8s.io/kops/upup/pkg/fi/cloudup/terraform"
+)
+
+// Disk represents a ALI Cloud Disk
+//go:generate fitask -type=Disk
+const (
+	DiskResource = "disk"
+	DiskType     = ecs.DiskTypeAllData
```
Comment (andrewsykim) [inline upup/pkg/fi/cloudup/alitasks/disk.go]: just curious, what does ecs stand for?

**Comment by andrewsykim** [comment]
/lgtm

Thanks @LilyFaFa!

File: upup/pkg/fi/cloudup/alitasks/disk.go (hunk 1 above)
Comment (xh4n3) [reply upup/pkg/fi/cloudup/alitasks/disk.go]: ecs stands for Elastic Computing Engine, the EC2 for AliCloud.
Most of the product APIs are implemented in ecs package.

**Pull request merged**